src/app/pages/ArticleScreen/components/ArticleScreen.styles.ts

```diff
@@ -0,0 +1,45 @@
+import { styled } from '@styles/themes/defaultTheme';
+import { Text } from '@styles/components';
+
+export const ArticleWrapper = styled.ScrollView`
+  width: 100%;
+  padding: 0 ${props => props.theme.padding.large};
+  background-color: ${props => props.theme.colors.primary};
+`;
+
+export const ArticleHeader = styled(Text)`
+  font-size: ${props => props.theme.font.size.xlarge};
+  color: ${props => props.theme.colors.secondary};
+  margin: ${props => props.theme.margin.small} 0
+    ${props => props.theme.margin.medium} 0;
+`;
+
+export const ArticleContentWrapper = styled.View``;
+
+export const ArticleMetaDataWrapper = styled.View`
+  flex-direction: row;
+  justify-content: space-between;
+  margin: ${props => props.theme.margin.medium} 0
+    ${props => props.theme.margin.small} 0;
+`;
+
+export const ArticleImage = styled.Image`
+  width: 100%;
+  height: 200;
```

Comment:
no units?

Reply:
I went with default react-native units which are [density-independent pixels](https://facebook.github.io/react-native/docs/height-and-width). Would you like me to change it on pixels?

Reply:
they can stay